plutus-tx-plugin/test/Plugin/Coverage/Spec.hs

```diff
@@ -44,9 +44,6 @@ fun x = case x of
 otherFun :: Integer -> Bool
 otherFun x = (x P.== 5) && True
 
-boolOtherFunctionSimplifiesAway :: CompiledCode (Integer -> Bool)
```

Comment:
Reason this test case is removed: it is no longer applicable when using `Strict`. The "simplifies away" here happened because GHC performed beta reduction, converting `\x -> otherFun x` to `otherFun`. With `Strict`, this beta reduction is no longer valid, so `otherFun` won't be simplified away.